Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -290,6 +290,16 @@
   categories:
     - UplinkUtility
 
+- type: listing
+  id: UplinkDimensionPot
+  name: uplink-dimension-pot-name
+  description: uplink-dimension-pot-desc
+  productEntity: DimensionPot
+  cost:
+    Telecrystal: 14
```

Needs to be 20 or higher
```suggestion
    Telecrystal: 20
```